lib/record_store/cli.rb

```diff
@@ -224,7 +226,8 @@ def validate_change_size
         end
 
         unless removals.empty?
-          abort "As a safety measure, you cannot remove more than #{MAXIMUM_REMOVALS} records at a time per zone. (zones failing this: #{removals.map(&:name).join(', ')})"
+          abort "As a safety measure, you cannot remove more than #{MAXIMUM_REMOVALS} "\
```

Comment:
Is this supposed to be aligned exactly the same way ?
Maybe you could use a temporary variable to store the string for readability:

```
error = +"As a safety measure, you cannot remove more than #{MAXIMUM_REMOVALS} "
error << 'records at a time per zone. '
error << "(zones failing this: #{removals.map(&:name).join(', ')})"
abort error
```